src/npm-publish.ts

```diff
@@ -35,6 +35,7 @@ export async function npmPublish(opts: Options = {}): Promise<Results> {
 
   let results: Results = {
     package: manifest.name,
+    registry: options.registry,
```

Comment:
I think this is a good change but can it be split into its own PR so this one can focus on getting the build fixed / deprecations removed?

Reply:
Been a while so I can't remember why I added it here. It should be exposed though so consumers can get the default / final value that is used.

Reply:
I remember now, it's because I was fixing something related to this output and to keep things consistent it would need to pull from `results` everywhere instead of `options`.

https://github.com/JS-DevTools/npm-publish/blob/71869b5f0195f8c289f623103692895dd046d50d/src/cli/index.ts#L37